### PR TITLE
1832/feat/dashboard redo

### PR DESCRIPTION
**Pull request by y-golde**
Dashboard redo.
see ticket for all changes

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/convertorsToGraph.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/convertorsToGraph.ts:1]: remove

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/convertorsToGraph.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import InvestigationChart from 'models/InvestigationChart';
+import FilterRulesDescription from 'models/enums/FilterRulesDescription';
+import { InvesitgationInfoStatistics } from 'models/InvestigationStatistics';
+
+
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/convertorsToGraph.ts:6]: remove

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx [hunk 3]
```diff
@@ -51,21 +33,31 @@ const InvestigationsInfo = (props: Props): JSX.Element => {
...
-                                    amountOfInvestigations={InvestigationData.value}
-                                    text={InvestigationData.id}
-                                    style={{ backgroundColor: InvestigationData.color }}
-                                    onClick={() => onInfoButtonClick(statusToFilterConvertor[InvestigationData.id], InvestigationData.id)}
-                                />
-                            ))
+                            investigationsGraphData.map((InvestigationData: InvestigationChart , index) => {
+                                return  (
+                                    <>
+                                    <Grid item xs={3}>
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx:41]: add indentation right

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx [hunk 4]
```diff
@@ -51,21 +33,31 @@ const InvestigationsInfo = (props: Props): JSX.Element => {
...
+                                            id={`info-button-${index}`}
+                                            key={`investigationInfoButton-${index}`}
+                                            amountOfInvestigations={InvestigationData.value}
+                                            text={InvestigationData.id}
+                                            style={{ backgroundColor: InvestigationData.color }}
+                                            onClick={() => onInfoButtonClick(statusToFilterConvertor[InvestigationData.id], InvestigationData.id)}
+                                        />
+                                    </Grid>
+                                    {InvestigationData.space &&  Array.from(Array(InvestigationData.space))
+                                        .map((_,i) => {
```
Comment (Noam6421) [inline client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx:52]: what is the _ ?

File: client/src/models/InvestigationStatistics.ts [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
     unallocatedInvestigations: number;
+    transferRequestInvestigations: number;
     unusualInProgressInvestigations: number;
     unusualCompletedNoContactInvestigations: number;
-    transferRequestInvestigations: number;
     waitingForDetailsInvestigations: number;
 }
+interface InvesitgationStatistics extends InvesitgationInfoStatistics {
+    allInvestigations: number;
+}
```
Comment (Noam6421) [inline client/src/models/InvestigationStatistics.ts:12]: ; at end

**Review by Noam6421 — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx (hunk 4 above)
Comment (y-golde) [reply client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx:55]: its ignore - a blank constant, Array.from(Array(number)) returns an empy array with width 'number' and i want to use the index

**Review by Noam6421 — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**